### PR TITLE
Integrations: rename Configure to Add to reflect current SW

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Integrations: rename Configure to Add to reflect current SW
image

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature others were adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/22387
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Release Notes

- **New Features**
	- Updated integration setup instructions across multiple integrations to improve clarity and user experience.
  
- **Documentation**
	- Changed terminology from "Configure" to "Add" in various integration documentation, including ecobee, MQTT, Z-Wave, and others, to reflect the updated user interface.
	- Enhanced formatting and readability of documentation for integrations such as WeMo, Roon, and Thread, including clearer tables and structured content.
  
- **Bug Fixes**
	- Improved clarity in troubleshooting steps for various integrations, ensuring users have accurate guidance for resolving issues.

### CODE REVIEW

File: source/_includes/integrations/option_flow.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ Options for {{ name }} can be set via the user interface, by taking the followin
 - Browse to your Home Assistant instance.
 - Go to **{% my integrations title="Settings > Devices & Services" %}**.
 - If multiple instances of {{ name }} are configured, choose the instance you want to configure.
-- Select the integration, then select **Configure**.
+- Select the integration, then select **Add**.
```
Comment (mib1185) [inline source/_includes/integrations/option_flow.md]: This should keep `**Configure**` since it is the options flow, not the config flow

**Review by mib1185 — CHANGES_REQUESTED** [review]

File: source/_includes/integrations/option_flow.md (hunk 1 above)
Comment (c0ffeeca7) [reply source/_includes/integrations/option_flow.md]: Oh, thank you for the catch!

File: source/_integrations/ecobee.markdown [hunk 2]
```diff
@@ -89,7 +89,7 @@ api_key:
   <img src='/images/screenshots/ecobee-thermostat-card.png' />
 </p>
 
-You must [restart Home Assistant](/docs/configuration/#reloading-changes) for the changes to take effect. After restarting, go to {% my integrations title="**Settings** > **Devices & services**" %} and select the integration. Then, select **Configure** and continue to authorize the app according to the above **Automatic Configuration**, starting at step 2.
+You must [restart Home Assistant](/docs/configuration/#reloading-changes) for the changes to take effect. After restarting, go to {% my integrations title="**Settings** > **Devices & services**" %} and select the integration. Then, select **Add** and continue to authorize the app according to the above **Automatic Configuration**, starting at step 2.
```
Comment (mib1185) [inline source/_integrations/ecobee.markdown:92]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/forecast_solar.markdown [hunk 3]
```diff
@@ -135,4 +135,4 @@ instance:
 1. Browse to your Home Assistant instance.
 2. Go to **{% my integrations title="Settings > Devices & services" %}**.
 3. If multiple instances of {{ name }} are configured, choose the instance you want to configure.
-4. Select **Configure**.
+4. Select **Add**.
```
Comment (mib1185) [inline source/_integrations/forecast_solar.markdown:138]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/fritz.markdown [hunk 4]
```diff
@@ -60,16 +60,16 @@ Set a new password for the guest wifi.
...
+| -------------- | -------- | ----------------------------------------------------- |
+| `device_id`    | no       | Only act on a specific  router                        |
+| `password`     | yes      | New password for the guest wifi                       |
+| `length`       | yes      | Length of the auto-generated password. (_default 12_) |
 
 ## Integration options
 
 It is possible to change some behaviors through the integration options.
-To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. Select the **AVM FRITZ!Box Tools** integration, then select **Configure**.
+To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. Select the **AVM FRITZ!Box Tools** integration, then select **Add**.
```
Comment (mib1185) [inline source/_integrations/fritz.markdown:72]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/insteon.markdown [hunk 5]
```diff
@@ -63,7 +63,7 @@ The Insteon configuration panel allows for product specific configuration of Ins
 To open the Insteon configuration panel:
 
 1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
-2. Select the **Insteon** {% term integration %}. Then, select **Configure** to open the Insteon configuration panel.
+2. Select the **Insteon** {% term integration %}. Then, select **Add** to open the Insteon configuration panel.
```
Comment (mib1185) [inline source/_integrations/insteon.markdown:66]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/lcn.markdown [hunk 6]
```diff
@@ -64,7 +64,7 @@ The `lcn` hardware modules and groups are represented by Home Assistant *devices
 
 The configuration of the `lcn` devices and entities is completely done using a web user interface (configuration panel).
 
-Once the integration is added to Home Assistant, you can access the `lcn` configuration panel by selecting the **Configure** button next to the respective integration entry on the [LCN integration page](https://my.home-assistant.io/redirect/integration/?domain=lcn).
+Once the integration is added to Home Assistant, you can access the `lcn` configuration panel by selecting the **Add** button next to the respective integration entry on the [LCN integration page](https://my.home-assistant.io/redirect/integration/?domain=lcn).
```
Comment (mib1185) [inline source/_integrations/lcn.markdown:67]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/fritz.markdown (hunk 4 above)
Comment (c0ffeeca7) [reply source/_integrations/fritz.markdown:72]: I'm glad you're more alert than me. Time to call it a day for me, it seems.

File: source/_integrations/mqtt.markdown [hunk 7]
```diff
@@ -147,7 +147,7 @@ Add the MQTT integration, then provide your broker's hostname (or IP address) an
 
 1. Go to **{% my integrations title="Settings > Devices & services" %}**.
 2. Select the MQTT integration.
-3. Select **Configure**, then **Re-configure MQTT**.
+3. Select **Add**, then **Re-configure MQTT**.
```
Comment (mib1185) [inline source/_integrations/mqtt.markdown:150]: I think this is also related to an already existing config entry, those related to the options flow?

File: source/_integrations/mqtt.markdown [hunk 8]
```diff
@@ -197,7 +197,7 @@ To change the settings, follow these steps:
...
```
Comment (mib1185) [inline source/_integrations/mqtt.markdown:200]: I think this is also related to an already existing config entry, those related to the options flow?

**Comment by mib1185** [comment]
i'm not done with the list, but I think there are more of these "I think this is also related to an already existing config entry, those related to the options flow?" 🙈

**Comment by c0ffeeca7** [comment]
> i'm not done with the list, but I think there are more of these "I think this is also related to an already existing config entry, those related to the options flow?" 🙈

yes, I see... let me start over, on Monday, with a fresh mind :laughing: Thank you for bearing with me, though.

**Comment by mib1185** [comment]
you're welcome 😁

**Pull request closed without merging**